docs/docs/contributors/developers-guide/maintainers.md

```diff
@@ -40,7 +40,7 @@ Mealie is published via GitHub actions to the GitHub container registry with the
 : published when a new GitHub Release is created - [Actions File](https://github.com/mealie-recipes/mealie/blob/mealie-next/.github/workflows/release.yml)
 
 !!! note
-    Both the latest, and {version} tags will be the same container on the release of a new version
+    Both the latest, and {version} tags will be the same image on the release of a new version (until the next nightly is published)
```

Comment:
Should latest not be the latest release? So nightly is not equal to latest.

Reply:
Brain fart after a long day.. Will remove the bit in brackets.

Reply:
Good catch. It made sense in my head as I wrote it.. I was trying to say there will be releases (on nightly) that are need than latest or {version}